store/driver/txn/snapshot.go

```diff
@@ -130,6 +130,11 @@ func (s *tikvSnapshot) SetOption(opt int, val interface{}) {
 		s.KVSnapshot.SetRequestSourceType(val.(string))
 	case kv.ReplicaReadAdjuster:
 		s.KVSnapshot.SetReplicaReadAdjuster(val.(txnkv.ReplicaReadAdjuster))
+	case kv.ScanBatchSize:
+		size := val.(int)
+		if size > 0 {
+			s.KVSnapshot.SetScanBatchSize(size)
```

Comment:
what‘s the use of this size

Reply:
`KVSnapshot.Iter` use `Scanner` to scan key/values from storage(TiKV), the `ScanBatchSize` uses to control to fetch how many kv in one RPC request.